poetry/masonry/builders/builder.py

```diff
@@ -91,14 +90,18 @@ def find_files_to_add(self, exclude_build=True):  # type: () -> list
                     " - Adding: <comment>{}</comment>".format(str(file)),
                     verbosity=self._io.VERBOSITY_VERY_VERBOSE,
                 )
-                to_add.append(file)
+                to_add.append((file, file))
+
+        _, scripts = self.convert_entry_points()
+        for name, script in scripts.items():
+            to_add.append((Path(script), Path("bin") / name))
```

Comment:
Also using `relative_to` like below would be good since it would fail on values like `"../other-project/bin/script"` - `convert_entry_points` will need to return absolute paths for scripts in order for this to behave as expected.